src/app/services/data/node.service.ts

```diff
@@ -256,6 +254,16 @@ export class NodeService implements OnDestroy {
     return node;
   }
 
+  addEmptyNode(): Node {
```

Comment:
Can we set this node as collapsed from the very start, before firing the "create" event? This could simplify the OSRD side.

Reply:
sure